test/contextstore.test.js

```diff
@@ -356,6 +356,30 @@ describe('Koa Session External Context Store', () => {
     });
   });
 
+  describe('when autoCommit is present and set to false', () => {
+    it('should not set headers if commit() isn\'t called', done => {
```

Comment:
need one more case to ensure when we call session.commit and koa-session will set cookie in headers